network/src/counters.rs

```diff
@@ -3,17 +3,40 @@
 
 use lazy_static;
 use libra_metrics::{Histogram, IntCounter, IntGauge, OpMetrics};
-use prometheus::IntGaugeVec;
+use prometheus::{IntCounterVec, IntGaugeVec};
 
 lazy_static::lazy_static! {
-    pub static ref NETWORK_PEERS: IntGaugeVec = register_int_gauge_vec!(
+    pub static ref LIBRA_NETWORK_PEERS: IntGaugeVec = register_int_gauge_vec!(
         // metric name
         "libra_network_peers",
         // metric description
         "Libra network peers counter",
         // metric labels (dimensions)
         &["state", "role"]
     ).unwrap();
+
+    pub static ref LIBRA_RPC_REQUESTS: IntCounterVec = register_int_counter_vec!(
+        "libra_rpc_requests",
+        "Libra rpc requests counters",
+        &["unit", "state"]
+    ).unwrap();
+
+    pub static ref LIBRA_RPC_RESPONSES: IntCounterVec = register_int_counter_vec!(
+        "libra_rpc_responses",
```

Comment:
How are `rpc_requests` and `rpc_responses` related? It may make sense for these to be the same metric name with a label.